lib/open_food_network/customers_report.rb

```diff
@@ -1,9 +1,10 @@
 module OpenFoodNetwork
   class CustomersReport
     attr_reader :params
-    def initialize(user, params = {})
+    def initialize(user, params = {}, compile_table = false)
       @params = params
       @user = user
+      @compile_table = compile_table
```

Comment:
while do you refer to `compile` and not `render`. What's the difference with the BulkCoopReport above?

Reply:
"render" is used to draw something on the screen. If you you have two `render` statements in a controller, it will raise an error. The intent here is not just to avoid the rendering of data, it's to avoid all the data loading and sorting in the first place.